core/src/consensus/tendermint/mod.rs

```diff
@@ -595,6 +603,7 @@ impl ConsensusEngine<CodeChainMachine> for Tendermint {
         let view = self.view();
 
         let (last_block_hash, last_block_view) = &*self.last_confirmed_view.read();
+        assert_eq!(last_block_hash, &parent.hash());
```

Comment:
This makes the test fail.
Can we ignore assert_eq! in the test environment?

@sgkim126 @jhs7jhs @foriequal0  Do you have any idea?

Reply:
It's possible to use `#[cfg(not(test))]` but I don't recommend.

Reply:
I found that I can fix this problem by making test client to use the Tendermint test scheme.
I've fixed now.